### PR TITLE
Added Multiple Pages 

### PR DESCRIPTION
**Pull request by ThenoobMario**
#9 

Added multiple pages with Firebase support to store user feedback, along with a Feedback.MD guide which walks through the process of setting up Firebase in the app. 

Also grouped the already existing guides into its own dedicated folder for easy navigation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,15 +116,15 @@ __NOTE: For now we are exclusively focused on image classification models built
 
       - Now as per your choice of hosting, please refer the following guides:
         - [Heroku [Recommended for Beginners]](./Heroku_Guide.MD)
-        - [Google Cloud Platform](./GKE_Guide.MD)
-        - [Microsoft Azure](./Microsoft_Azure_Guide.MD)
+        - [Google Cloud Platform](.Guides/GKE_Guide.MD)
+        - [Microsoft Azure](.Guides/Microsoft_Azure_Guide.MD)
```
Comment (smaranjitghose) [inline README.md]: The path would be  starting with ``./``

File: app.py [hunk 2]
```diff
@@ -54,11 +57,45 @@ def display_team(name,path,affiliation="",email=""):
...
             team_img = Image.open(path)
+
             st.image(team_img, width=350, use_column_width=False)
             st.markdown(f"## {name}")
             st.markdown(f"#### {affiliation}")
             st.markdown(f"###### Email {email}")
             st.write("------")
+
         display_team("Your Awesome Name", "./assets/profile_pic.png","Your Awesome Affliation","hello@youareawesome.com")
+
```
Comment (smaranjitghose) [inline app.py:69]: Nice work!

File: feedback_db.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+def feedback_db(user_email, feedback):
+    # Setting the Firebase configurations of the App
+    # Check Feedback.MD to learn how to get the information
+    config = {
+    "apiKey": "apiKey",
+    "authDomain": "projectId.firebaseapp.com",
+    "databaseURL": "https://databaseName.firebaseio.com",
+    "storageBucket": "projectId.appspot.com"
+    }
+
```
Comment (smaranjitghose) [inline feedback_db.py]: Transfer this to the file I sent you over gitter

File: send_img.py [hunk 4]
```diff
@@ -1,21 +1,21 @@
-from firebase import Firebase
+import pyrebase
 
+# Configuration Information
+config = {
+"apiKey" :                   , 
+"authDomain" :        ,
+"databaseURL" :      ,
+"projectId":         , 
+"storageBucket" :     ,
```
Comment (smaranjitghose) [inline send_img.py]: This file is no more required

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (ThenoobMario) [reply app.py:69]: Thank you 😃

**Comment by ThenoobMario** [comment]
Made all the changes which were asked for and also resolved the conflicts.

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
give @ThenoobMario 75 points

**Pull request merged**